### PR TITLE
feat(APIv2): RHINENG-13307 create tailorings via API explicitly

### PR DESCRIPTION
**Pull request by skateman**
This is a feature that ImageBuilder requested, they need to be able to create "empty" policies without systems.

## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [x] General Coding Practices

### CODE REVIEW

File: spec/api/v2/schemas.rb [hunk 1]
```diff
@@ -43,6 +43,7 @@ module Schemas
         supported_profile: SUPPORTED_PROFILE,
         system: SYSTEM,
         tailoring: TAILORING,
+        create_tailoring: CREATE_TAILORING,
```
Comment (romanblanco) [inline spec/api/v2/schemas.rb:46]:
```suggestion
        created_tailoring: CREATED_TAILORING,
```

Comment (skateman) [reply spec/api/v2/schemas.rb:46]: Semantically not correct, it's the schema that is needed when you create a tailoring. Not for a tailoring that was already createD.

Comment (romanblanco) [reply spec/api/v2/schemas.rb:46]: `create_tailoring` does not seem fitting, as it describes more the action, which is not what is described in scheme.

What about `new_tailoring`?

Comment (skateman) [reply spec/api/v2/schemas.rb:46]: It's not a new, it is connected to the creation.

Comment (skateman) [reply spec/api/v2/schemas.rb:46]: What's wrong with CREATE_TAILORING?

Comment (skateman) [reply spec/api/v2/schemas.rb:46]: I might consider swapping it to TAILORING_CREATE.

Comment (skateman) [reply spec/api/v2/schemas.rb:46]: I think that would be consistent with `policy_update` so I'll do that.

File: spec/integration/v2/tailorings_spec.rb [hunk 2]
```diff
@@ -91,6 +91,35 @@
...
+        FactoryBot.create(
+          :v2_policy,
+          account: user.account,
+          profile: FactoryBot.create(:v2_profile, ref_id_suffix: 'foo', supports_minors: [1])
+        ).id
+      end
+
+      v2_auth_header
+      tags 'Policies'
+      description 'Create a Tailoring with the provided attributes (for ImageBuilder only)'
```
Comment (romanblanco) [inline spec/integration/v2/tailorings_spec.rb:106]: Minor: considering a reference to ImageBuilder 

```suggestion
      description 'Create a Tailoring with the provided attributes (for <a href="https://console.redhat.com/insights/image-builder">ImageBuilder</a> only)'
```

**Review by romanblanco — APPROVED** [review]
LGTM :+1:

File: spec/integration/v2/tailorings_spec.rb (hunk 2 above)
Comment (skateman) [reply spec/integration/v2/tailorings_spec.rb:106]: Not sure if that's a good idea...

**Pull request merged**